src/ui/marker.js

```diff
@@ -604,4 +607,29 @@ export default class Marker extends Evented {
     getPitchAlignment() {
         return this._pitchAlignment;
     }
+
+    /**
+     * Sets the current `color` property of the marker.
```

Comment:
I think it's better to phrase it consistent with the other setters, so mentioning setting the color of the default marker instead of the color property. ie. "Sets the color of the Marker if using a default marker" or something like that.

Reply:
Fixed.